src/bin/pgcopydb/main.c

```diff
@@ -43,6 +43,14 @@ static void log_semaphore_unlink_atexit(void);
 int
 main(int argc, char **argv)
 {
+	/* create a new process group and set current process as its leader.  This
+ 	 * allows the process group to be easily controlled, without affecting any
```

Comment:
Nitpicking: please conform to the comment style used in the rest of the project. It would then look like this:

``` C
    /*
     * Create a new process group...
     * 
```

Reply:
Done.